app/mailers/communicart_mailer.rb

```diff
@@ -27,6 +27,18 @@ def notification_for_approver(to_email, approval, alert_partial=nil)
     )
   end
 
+  def notification_for_approver_removed(to_email, approval, alert_partial=nil)
+    @approval = approval
+    @alert_partial = alert_partial
+    proposal = approval.proposal
+    send_proposal_email(
+      from_email: user_email_with_name(proposal.requester),
+      to_email: to_email,
+      proposal: proposal,
+      template_name: 'proposal_notification_email'
+    )
+  end
```

Comment:
Should there be a template or something associated with this email? Right now it's identical to `notification_for_approver`